### PR TITLE
Add definitions for Hispanic, Latino and Latinx

### PR DESCRIPTION
**Pull request by kgcreative**
First pass at Latino/Latina/Latinx/Hispanic. Latinx self-identification is a complex subject, so other voices would be welcome to revise and expand.

### CODE REVIEW

File: 11ty/definitions/hispanic.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
-defined: false
----
\ No newline at end of file
+defined: true
+reading:
+    - text: "Remezcla: Hispanic vs. Latino vs. Latinx: A Brief History of How These Words Originated"
+      href: https://remezcla.com/features/culture/latino-vs-hispanic-vs-latinx-how-these-words-originated/
+---
+
+of Spanish, or Spanish-speaking origin.
```
Comment (tatianamac) [inline 11ty/definitions/hispanic.md]: Add here: It is considered a pan-ethnic term and does not provide context for someone's race or country of origin.

**Comment by tatianamac** [comment]
@kgcreative Are you still interested in making these suggested edits so we may close out this PR?

File: 11ty/definitions/latino.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Latin American/Latino/Latina
+slug: latino
+defined: true
+reading:
+    - text: "Remezcla: Hispanic vs. Latino vs. Latinx: A Brief History of How These Words Originated"
+      href: https://remezcla.com/features/culture/latino-vs-hispanic-vs-latinx-how-these-words-originated/
+---
+
+From Latin America, or of Latin American origin.
```
[the remark this answers is not in the document]
Comment (kgcreative) [reply 11ty/definitions/latino.md]: I've updated this definition per your suggestions

File: 11ty/definitions/hispanic.md (hunk 1 above)
Comment (kgcreative) [reply 11ty/definitions/hispanic.md]: I've updated this definition, along with Latinx to call out the pan-ethnic nature of this term

**Comment by kgcreative** [comment]
@tatianamac updated the pull request with edits!

**Pull request merged**